GetCanvasTODOList/dcoder_run.yml

```diff
@@ -0,0 +1,9 @@
+commands:
+  - run: |
+      # Write your command here
+    default: true
+  - run: |
```

Comment:
Run this command using Dcoder_run or dcoder-cli to create a build in single file in dist folder.